.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: groovy
+jdk:
+- oraclejdk8
```

Comment:
I'd also recommend using a more recent version of JDK 8, like we do in the [JUnit 5 build](https://github.com/junit-team/junit5/blob/master/.travis.yml).